readthedocs/analytics/models.py

```diff
@@ -22,20 +22,18 @@ def _last_30_days_iter():
 
 
 class PageViewManager(models.Manager):
-    def register_page_view(self, project, version, path, status):
-        # Normalize path to avoid duplicates.
-        path = path.strip("/")
-        if not path:
-            path = "/"
-        fields = dict(
+    def register_page_view(self, project, version, path, full_path=None, status=200):
+        # Normalize paths to avoid duplicates.
+        path = path.strip("/") or "/"
+        full_path = full_path.strip("/") or "/"
```

Comment:
Also also, we are currently saving the relative paths without an `/`, so new page views will be counted separately from previous ones (but this should be normalized after some days by itself, since we remove old page views).